fedimint-api/src/db/mod.rs

```diff
@@ -169,10 +175,99 @@ impl Drop for CommitTracker {
     }
 }
 
+/// IsolatedDatabaseTransaction is a wrapper around DatabaseTransaction that is responsible for
```

Comment:
nit: using inline code formatting for type names would look nicer in rendered docs